pseudocode.md

```diff
@@ -1,13 +1,66 @@
 # Pseudocode
```

Comment:
I saw you didn't have comments yet and I only had 2 people with a pr, so I thought I'd give you feedback -- not sure how much i'll truly find tho :D

Reply:
I'm honored, you clearly found enough from the looks of things. 😄